src/examples/hello_world/hello_world.ts

```diff
@@ -0,0 +1,33 @@
+import {
+  Field,
+  SmartContract,
+  state,
+  State,
+  Party,
+  method,
+  DeployArgs,
+  PrivateKey,
+} from '../../../dist/server';
+
+export const adminPrivateKey = PrivateKey.random();
+export const adminPublicKey = adminPrivateKey.toPublicKey();
+
+export class HelloWorld extends SmartContract {
+  @state(Field) x = State<Field>();
+
+  deploy(input: DeployArgs) {
+    super.deploy(input);
+    this.x.set(Field(3));
+    Party.setValue(this.self.update.delegate, adminPublicKey);
```

Comment:
What does `this.self.update.delegate` refer too and why are we setting it to `adminPublicKey`?

Reply:
This line and the deploy method is from the Hello World example presentation. @bkase is there a reason you added this to the presentation example?

Reply:
This was added to the presentation to show off the account preconditions features. It’s probably nice to mirror the presentation so I think we should leave it in